db_pyt.py

```diff
@@ -0,0 +1,103 @@
+import sqlite3
+from typing import List, Any, Type
+from dataclasses import dataclass
+
+
+@dataclass
+class User:
+    id: int
+    name: str
+    email: str
+
+
+class DatabaseTable:
+    def __init__(self, db_name: str, table_name: str, user_class: Type):
+        self.db_name = db_name
+        self.table_name = table_name
+        self.user_class = user_class
+        self.create_table()
+
+    def create_table(self):
+        with sqlite3.connect(self.db_name) as conn:
+            cursor = conn.cursor()
+            cursor.execute(f'''
+                CREATE TABLE IF NOT EXISTS {self.table_name} (
+                id INTEGER PRIMARY KEY AUTOINCREMENT,
+                name TEXT NOT NULL,
+                email TEXT NOT NULL UNIQUE
+                )
+            ''')
+            conn.commit()
+
+    def add_records(self, *records: Any):
+        with sqlite3.connect(self.db_name) as conn:
+            cursor = conn.cursor()
+            for record in records:
+                if isinstance(record, self.user_class):
+                    cursor.execute(f'''
```

Comment:
Here I'd better use execute_many as adding all records at once should be more efficient than adding one by one